wd_helper.au3

```diff
@@ -1303,6 +1304,12 @@ Func _WD_UpdateDriver($sBrowser, $sInstallDir = Default, $bFlag64 = Default, $bF
 		$iErr = @error
 
 		If $iErr = $_WD_ERROR_Success Then
+			If StringInStr($sBrowser, '\') Then 
+				; Extract filename from full path
+				$sBrowser = StringRegExpReplace($sBrowser, "^.*\\|\..*$", "")
+			EndIf
+
+			; Match in list of supported browsers
 			Local $iIndex = _ArraySearch($_WD_SupportedBrowsers, $sBrowser, Default, Default, Default, Default, Default, $_WD_BROWSER_Name)
```

Comment:
What do you think about having _WD_GetBrowserVersion set @extended to the matching index (obviously only when the function is successful)? That would eliminate the need to perform this extraction again or even perform the _ArraySearch again.

Reply:
Good point.
I'll make the appropriate changes

Reply:
Looks good to me